client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -0,0 +1,267 @@
+/**
+ * VIAME CSV parser/serializer copied logically from
+ * viame_server.serializers.viame python module
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import csvparser from 'csv-parse';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
The comment is all over the place, but I only want to do this in `platform/desktop` because I don't want any desktop-only dependencies to pollute the dependency list of `vue-media-annotator` and we are never going to expose desktop code as an NPM library, so this abuse won't affect anyone but us.

I'd be OK with that rule as long as it goes in `platform/desktop/.eslintrc` instead of package.json